### ISSUE TITLE
Visual Presentation

### ISSUE THREAD

**Issue by NovaStellar-DunEDU** [NovaStellar-DunEDU/RockBands#4]
Add more spacing for the Add Albumship page, and change the name of the menu button to be "Assign Album to Band"

This website fails the 1.5-line spacing condition from the WCAG 2 checklist.
1.4.8, Visual Presentation
